metrics/ethereum.py

```
@@ -175,31 +181,41 @@ def __init__(
+        websocket: websockets.WebSocketClientProtocol,
+        timeout: float,
+    ) -> str:
         """Receive a message with a timeout."""
         try:
             message = await asyncio.wait_for(websocket.recv(), timeout)
             # Log incoming message size in bytes
             message_size: int = len(message.encode("utf-8"))
             logging.info(
-                f"WebSocket received {message_size} bytes from {self.labels.get_prometheus_labels()}"
+                f"WebSocket received {message_size} bytes from "
+                f"{self.labels.get_prometheus_labels()}"
```

Ruff (0.15.9)

[warning] Logging statement uses f-string

(G004)